yarn.lock

```diff
@@ -8355,44 +8103,23 @@ ember-composable-helpers@^4.4.1:
     "@glimmer/env" "^0.1.7"
     broccoli-merge-trees "^4.2.0"
     ember-cli-babel "^7.18.0"
-    ember-cli-typescript "^3.1.3"
-    ember-inflector "^3.0.1"
-
-"ember-data-latest@npm:ember-data@latest":
-  version "3.27.1"
-  resolved "https://registry.yarnpkg.com/ember-data/-/ember-data-3.27.1.tgz#02d6b76c1d5de0f50d24274147b8711f6ca4f28d"
-  integrity sha512-36P8+7B6Z5ZjyITFbf2Wcub/fdE2DTsLoPPZK7It488fub5s90o85XC0WlwUQPvff39us2N4pzjwmCZ8Jj/gjg==
-  dependencies:
-    "@ember-data/adapter" "3.27.1"
-    "@ember-data/debug" "3.27.1"
-    "@ember-data/model" "3.27.1"
-    "@ember-data/private-build-infra" "3.27.1"
-    "@ember-data/record-data" "3.27.1"
-    "@ember-data/serializer" "3.27.1"
-    "@ember-data/store" "3.27.1"
-    "@ember/edition-utils" "^1.2.0"
-    "@ember/ordered-set" "^4.0.0"
-    "@ember/string" "^1.0.0"
-    "@glimmer/env" "^0.1.7"
-    broccoli-merge-trees "^4.2.0"
-    ember-cli-babel "^7.26.6"
-    ember-cli-typescript "^4.1.0"
-    ember-inflector "^4.0.1"
+    ember-cli-typescript "^3.1.3"
+    ember-inflector "^3.0.1"
 
-ember-data@~3.28.0:
-  version "3.28.0"
-  resolved "https://registry.yarnpkg.com/ember-data/-/ember-data-3.28.0.tgz#aa0f104bfdfb27b7b9b8e7ccf722f120ae6cd68b"
-  integrity sha512-IZiqO/GL7lvt63qCD5dLeeeUQkwPj4vUfxoIks1d2BEBFYYLG6LxIkJ+qQIKvmk1f5r644hQo6KrWgcP69A+WA==
-  dependencies:
-    "@ember-data/adapter" "3.28.0"
-    "@ember-data/debug" "3.28.0"
-    "@ember-data/model" "3.28.0"
-    "@ember-data/private-build-infra" "3.28.0"
-    "@ember-data/record-data" "3.28.0"
-    "@ember-data/serializer" "3.28.0"
-    "@ember-data/store" "3.28.0"
+"ember-data-latest@npm:ember-data@latest", ember-data@~3.28.0:
```

Comment:
I believe this trips up yarn, and yarn ends-up onyl installing `node_modules/ember-data-latest` rather then both `node_modules/ember-data` and `node_modules/ember-data-latest`. Even though they are the same version, they have a different name and should not be collapsed into one, as that breaks the node resolution algorithm.

Reply:
Fixed by: https://github.com/embroider-build/embroider/pull/951#discussion_r709561955